src/website/views/templates/autopay.pug

```diff
@@ -0,0 +1,592 @@
+                      <option>13</option>
+                      <option>14</option>
+                      <option>15</option>
+                      <option>16</option>
+                      <option>17</option>
+                      <option>18</option>
+                      <option>19</option>
+                      <option>20</option>
+                      <option>21</option>
+                      <option>22</option>
+                      <option>23</option>
+                      <option>24</option>
+                      <option>25</option>
+                      <option>26</option>
+                      <option>27</option>
+                      <option>28</option>
+                      <option>29</option>
+                      <option>30</option>
+                      <option>31</option>
+                    </select><select class="a-input -xl -mr-md--1" id="expiration">
+                      <option>2019</option>
+                      <option>2020</option>
+                      <option>2021</option>
+                      <option>2022</option>
+                      <option>2023</option>
+                      <option>2024</option>
+                      <option>2025</option>
+                      <option>2026</option>
+                      <option>2027</option>
+                      <option>2028</option>
+                      <option>2029</option>
+                      <option>2030</option>
+                    </select></div>
+                  </div>
+                </div>
+                <div class="a-col -w-md--4 -w-sm--12">
+                  <div class="m-formGroup -labeled">
+                    <label class="a-label" for="zip">Billing zip code</label>
+                    <input class="a-input -xl" type="text" id="zip" placeholder="">
+                  </div>
+                </div>
+              </div>
+            </fieldset>
+          </div>
+        </div>
+      </div>
+    </div>
+
+h4 Optional
+.example.-mb--3.-bg--grey-10
+  .chi-app.-p3
+    .m-epanel.-bordered.-p--1#autopayOptional
+      .m-epanel__header.-d--flex.-justify-content--between
+        div
+          h5.-m--0.a-card__title Automatic Payments
+          p.-text Each month, an automatic payment for the full amount of your bill will be paid to CenturyLink from the payment method of your choosing.
+        .m-formGroup
+          input(type="checkbox", class="a-input -toggle", id="toggle1", data-chi-epanel-action='toggle')
+          label(for="toggle1")
+      .m-epanel__collapse
+        .-active--only
+          .m-epanel__body
+            .m-epanel__content.-p--0
+              .a-grid.-noGutter
+                div.a-col.-w-md--4
+                  .m-formGroup.-labeled.-mb--1
+                    label.a-label(for='Optional') Choose a payment method
+                    .m-dropdown
+                      button.a-btn.-xl.m-dropdown__trigger(id='Optional', onclick='selectPayment(id)') Checking Account
```

Comment:
Label casing is inconsistent. Change "Checking Account" to "Checking account"

Reply:
done